apps/site/components/Common/Searchbox/Chat/ChatInput/index.module.css

```diff
@@ -0,0 +1,85 @@
+.promptWrapperCustom {
+  @apply relative;
+  @apply flex;
+  @apply items-center;
+  @apply rounded-2xl;
+  @apply bg-white;
+  @apply text-base;
+  @apply shadow-sm;
+
+  background-color: #1a1a1a !important;
+  border: 1px solid #2c3437 !important;
+  color: #fff !important;
+  margin: 0 var(--spacing-l, calc(16rem / var(--orama-base-font-size, 16))) !important;
+}
+
+.promptFieldCustom {
+  @apply w-full;
+  @apply resize-none;
+  @apply border-0;
+  @apply px-0;
+  @apply py-0;
+  @apply text-white;
+  @apply placeholder:text-neutral-400;
+  @apply focus:outline-none;
+
+  background: transparent !important;
```

Comment:
This, and other things, can be Tailwind (i.e. `bg-transparent`). If you want it to be `!important`, prefix with `!`, (i.e. `!bg-transparent`)